DirectProgramming/Fortran/CombinationalLogic/openmp-primes/sample.json

```diff
@@ -3,23 +3,23 @@
   "name": "OpenMP* Primes",
   "categories": ["Toolkit/oneAPI Direct Programming/Fortran/Combinational Logic"],
   "description": "Fortran Tutorial - Using OpenMP* (OMP)",
-  "toolchain": [ "ifort" ],
+  "toolchain": [ "ifx" ],
   "languages": [ { "fortran": {} } ],
   "targetDevice": [ "CPU" ],
   "os": [ "darwin" ],
   "builder": [ "make" ],
   "ciTests":{
     "darwin": [
```

Comment:
@krisdale Can you add the build steps for Linux and Windows? Remove ``darwin`` and add ``linux`` and ``windows``.